sci-mathematics/mathematica/mathematica-13.0.1.ebuild

```diff
@@ -0,0 +1,140 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CHECKREQS_DISK_BUILD=19G
+inherit desktop unpacker xdg
```

Comment:
For `CHECKREQS_DISK_BUILD` to do anything we do also need to inherit the `check-reqs` eclass

Reply:
Done, added check-reqs as inherit